### DISCUSSION TITLE
Academics Page UI

### DISCUSSION THREAD

**Discussion by Aaditya18SP** [rushiii3/KETS#29] [Ideas]
![IMG_20240525_225647](https://github.com/rushiii3/KETS/assets/107549181/07edf2b8-1c62-4f5d-b5b7-74a8a40207ca)
![IMG_20240525_225612](https://github.com/rushiii3/KETS/assets/107549181/f05b4435-d4b9-4684-921d-94ddec383755)
![IMG_20240525_225603](https://github.com/rushiii3/KETS/assets/107549181/08fc3d10-bde1-46e3-81f0-9004548eebc7)
![IMG_20240525_225551](https://github.com/rushiii3/KETS/assets/107549181/61ac3fcd-9174-4c29-97b6-cb84ab249314)

**Comment by AniketSingh52** [comment]
[Course_catalog_wireframe.pdf](https://github.com/rushiii3/KETS/files/15444394/Course_catalog_wireframe.pdf)
[Dept_wireframe.pdf](https://github.com/rushiii3/KETS/files/15444395/Dept_wireframe.pdf)
